powerhome/playbook
# [PLAY-962] Multi Level Select - Single Select Variant

**What does this PR do?**
- [Runway](https://nitro.powerhrg.com/runway/backlog_items/PLAY-962) - This is only part one: add radio buttons instead of checkboxes
- [Alpha](https://github.com/powerhome/nitro-web/pull/35016)
- Adds a `single` variant to the Multi Level Select kit
  - Only one option should ever be selected
  - If there is a default selection via `selectedIds`, and there are with multiple `id`s in the array, the first `id` is selected as the default, and the rest ignored
- Adds a new prop, `inputName`, which is required for a radio group so that only one option can be selected 
- Address some linting errors and warnings

**How to test?**
1. Go to the Playbook docs and see the new "Single Select" variant
2. See Runway story for Nitro testing

#### Checklist:
